typhos/widgets.py

```diff
@@ -393,6 +393,18 @@ def unit_changed(self, new_unit):
         if new_unit.lower() in EXPONENTIAL_UNITS and default:
             self.displayFormat = DisplayFormat.Exponential
 
+    @Property(bool, "dynamicFontSize")
```

Comment:
This isn't really used and probably should be removed
I erroneously thought that `TyphosLabel` et al were in the designer, but these are really used exclusively for auto-generated device displays (where we have the ophyd signal at instantiation time, etc)